honeybadgermpc/mpc.py

```diff
@@ -42,9 +42,9 @@ def __init__(self, sid, n, t, myid, pid, send, recv, prog, config, **prog_args):
         self.prog = prog
         self.prog_args = prog_args
 
-        # A task representing the opened values
-        # { shareid => Future (field list(field)) }
-        self._openings = {}
+        # Counter that increments by 1 every time you access it
```

Comment:
Um, this change seems to be there in both PRs, I will review this once the other one gets in - to get a better diff.